score_file_reader.py

```diff
@@ -0,0 +1,22 @@
+from game_score import GameScore
+from game_score_manager import GameScoreManager
+
+
+# Class to read a file with the leaderboard players/scores and store in a gameScoreManager
+class ScoreFileReader:
+    # input_file = file object containing the file with the leaderboard score data to read from
+    def __init__(self, input_file):
+        self.scoreFile = input_file
+
+    # Function to read the leaderboard score data from a file and store it in a GameScoreManager object
+    def read_scores(self):
+        # Get the lines
```

Comment:
I mainly added it since storing it in a GameScoreManager object wasn't completely apparent from the name, it could just read the file and return a string or something